linux/guest_customization/linux_gosc_workflow.yml

```diff
@@ -82,15 +82,23 @@
     # Check cloud-init version for cloud-init gosc workflow
     - include_tasks: ../utils/cloudinit_pkg_check.yml
 
-    # Skip cloud-init GOSC if cloud-init version is lower than 18.5
-    - include_tasks: ../../common/skip_test_case.yml
-      vars:
-        skip_msg: >
-          "The guest OS doesn't have cloud-init (> 18.5) installed.
-          Skip to run this test case."
-      when:
-        - enable_cloudinit_gosc
-        - (not cloudinit_version) or (cloudinit_version is version('18.5', '<='))
+    # Skip cloud-init GOSC if cloud-init version is lower than required version.
+    - block:
+        # Debian 10 and 11 cloud-init GOSC depends on cloud-init 22.1 or later
```

Comment:
Is it > 22.1 or > 18.5? from the comments, it seems = 22.1 or =18.5 is included.

Reply:
22.1 is included for Debian, 18.5 is excluded for the other OS.